### PR TITLE
STU-1225 add translation

### PR DESCRIPTION
**Pull request by lucaatstudylydotcom**
add translation

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,90 +3,90 @@
...
-	"ckeditor5-plugin",
-	"ckeditor5-math",
-	"katex"
+    "ckeditor",
+    "ckeditor5",
+    "ckeditor 5",
+    "ckeditor5-feature",
+    "ckeditor5-plugin",
+    "ckeditor5-math",
+    "katex"
```
Comment (MartinSchmidtStudyly) [inline package.json:12]: How do we have both an `package` as well as a `yarn` file?

File: src/ui/shortcutsview.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import View from '@ckeditor/ckeditor5-ui/src/view';
 import { groupBy } from 'lodash';
 import svgIcons from '../../assets/math-shortcut-icons';
 import global from "@ckeditor/ckeditor5-utils/src/dom/global";
-
+//TODO: language with t('')
```
Comment (MartinSchmidtStudyly) [inline src/ui/shortcutsview.js:6]: Was this TODO forgotten or for later implementation?

File: lang/translations/en.po [hunk 3]
```diff
@@ -0,0 +1,33 @@
+msgid ""
+msgstr ""
+"Language: \n"
+"Language-Team: \n"
+"Plural-Forms: \n"
```
Comment (MartinSchmidtStudyly) [inline lang/translations/en.po:5]: Same as `ckeditor5-image-max-width` plugin.

File: package.json (hunk 1 above)
Comment (lucaatstudylydotcom) [reply package.json:12]: i think the yarn is for the demo

File: src/ui/shortcutsview.js (hunk 2 above)
Comment (lucaatstudylydotcom) [reply src/ui/shortcutsview.js:6]: this is for later implementation (shortcutsview needs to be rewritten)

File: lang/translations/en.po (hunk 3 above)
Comment (lucaatstudylydotcom) [reply lang/translations/en.po:5]: same for me

**Review by MartinSchmidtStudyly — APPROVED** [review]

**Pull request merged**